### PR TITLE
Added tlg0592.tlg003.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Hermogenes Περὶ εὑρέσεως

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |      +45 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |   +20513 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0592/__cts__.xml`                          | New      |
| `./data/tlg0592/tlg003/__cts__.xml`                   | New      |
| `./data/tlg0592/tlg003/tlg0592.tlg003.1st1K-grc1.xml` | New      |
| `./data/tlg0592/__cts__.xml`                          | Passing  |
| `./data/tlg0592/tlg003/__cts__.xml`                   | Passing  |
| `./data/tlg0592/tlg003/tlg0592.tlg003.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3060)*

File: data/tlg0592/__cts__.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0592">
+  <ti:groupname xml:lang="lat">Hermogenes</ti:groupname>
+</ti:textgroup>
```
Comment (annettegessner) [inline data/tlg0592/__cts__.xml]: I think this file (now) already exists.

File: data/tlg0592/tlg003/tlg0592.tlg003.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,4377 @@
...
+  <title xml:lang="lat">Hermogenis Opera</title>
+  <editor>
+	<persName>
+	 <name>Hugo Rabe</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0592" xml:lang="lat">Hermogenes</author>
+  <imprint>
+	<publisher>Teubner</publisher>
+	<pubPlace>Leipzig</pubPlace>
```
Comment (annettegessner) [inline data/tlg0592/tlg003/tlg0592.tlg003.1st1K-grc1.xml]: I guess you are right and we don't need a German language xml tag before every German town's name.

**Comment by sonofmun** [comment]
@annettegessner Yes, we only need German language tags when the text of the tag is not a word or name in English. Since we call Leipzig "Leipzig" in English, we don't need it. If, however, we had "Köln", then we would need it.
And as long are there is no merge conflict with the file that already exists, that is no problem. This file needs to be there in the branch as well, otherwise it will not test with HookTest correctly.

**Pull request merged**